CHANGELOG.md

```diff
@@ -18,6 +24,7 @@
 
 ## [1.0.0.0] - 2018-05-10
 
+[1.2.0.0]: https://github.com/f-me/dmcc/compare/1.1.0.0...1.2.0.0
```

Comment:
I don't think we need 1.2.0.0 here – https://pvp.haskell.org/

Reply:
Would `1.1.0.1` be okay then?

Reply:
To be honest I don't know why it needed a version bump and Hackage release at all – looks like nothing really changed in the code or package dependencies (https://hackage.haskell.org/package/dmcc-1.1.0.1). If you need a fresh resolver for building locally from this repo just use current master now.

Reply:
Apart from anything else I think we urgently have to add version boundaries of dependencies to the `package.yaml` because a Hackage package doesn't know anything about stackage snapshots and which versions are compatible with this package. And then a fresh release would be a good idea, don't you think?

Reply:
Bounds are added automatically upon doing `stack sdist` thanks to `pvp-bounds: upper` (https://www.fpcomplete.com/blog/2015/09/stack-pvp)

Reply:
I've pushed a new release to Hackage but I really shouldn't have – it doesn't change anything apart from Stack resolver, which doesn't matter outside Stackage ecosystem anyways. Nothing changes since last "release" on Nov 4.

Reply:
Now I see those bounds [on hackage](https://hackage.haskell.org/package/dmcc), didn't know about that feature. Bounds look weird since there's no minimal version constraint, but it's better than nothing.

Reply:
@dzhus It looks like you pushed it with wrong resolver, now I see nightly snapshot in `master` branch, also got conflicts. Let me finish this PR first, then publish it after that.